src/main/java/com/revature/model/Pokemon.java

```diff
@@ -1,18 +1,156 @@
 package com.revature.model;
 
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name="Pokemon")
 public class Pokemon {
 
+	@Id
+	@Column(name="nationalid")
 	private int nationalid;
 	
+	@Column(name="name", nullable = false)
 	private String name;
 	
-	private String category;
-	
+	@Column(name="type1", nullable = false)
 	private String type1;
 	
+	@Column(name="type1")
 	private String type2;
 	
+	@Column(name="ability", nullable = false)
 	private String ability;
 	
+	@Column(name="category", nullable = false)
+	private String category;
```

Comment:
Not quite sure what this is

Reply:
Category is the type of pokemon like how sceptile is the forest pokemon